proc/api.thrift

```diff
@@ -0,0 +1,40 @@
+service Proc {
+  
+  exception InvalidRequest {
+    1: optional string message
+  }
+
+  exception ScootTooBusy {
+    1: optional i64 retryAfterMs
+  }
+  
+  enum JobType {
+    IRON_TESTS=1,
+    DEFAULT=2,
+  }
+
+  struct Command {
+    1: list<string> cmds
```

Comment:
Well yes, but what does that list represent? It says cmds, plural, so it must split things at the command level, not at the arg level. I'm floating the idea that it should be list<list<string>>.